utils.py

```diff
@@ -47,11 +51,20 @@ def getProdIP():
     return str(dns.resolver.query("cloud.redhat.com")[0])
 
 
-def getFlatData(data):
+def getFlatData(main_data, supplemental_data):
     ret = []
-    for key in data:
-        for val in data[key]:
-            ret.append((key, val))
+    # For every app that has frontend paths
```

Comment:
I think this could be simplifies using a lib like pylodash
https://pypi.org/project/pylodash/